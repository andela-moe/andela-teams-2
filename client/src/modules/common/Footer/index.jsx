import React from 'react';

export default () => (
  <footer className="footer-wrapper">
    <div className="col s12 white">&copy; {new Date().getFullYear()}</div>
  </footer>
);
