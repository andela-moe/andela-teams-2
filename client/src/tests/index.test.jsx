import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../modules/common/Footer';

describe('<Test Component />', () => {
  it('should mount without crashing', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
