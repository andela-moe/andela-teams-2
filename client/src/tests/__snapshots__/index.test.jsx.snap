// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Test Component /> should mount without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="col s12 white"
      >
        © 
        2018
      </div>,
      "className": "footer-wrapper",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "© ",
          2018,
        ],
        "className": "col s12 white",
      },
      "ref": null,
      "rendered": Array [
        "© ",
        2018,
      ],
      "type": "div",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="col s12 white"
        >
          © 
          2018
        </div>,
        "className": "footer-wrapper",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "© ",
            2018,
          ],
          "className": "col s12 white",
        },
        "ref": null,
        "rendered": Array [
          "© ",
          2018,
        ],
        "type": "div",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
